.changelog/16009.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+acl: Warn when query token parameter is used for authentication.
```

Comment:
What do you think about including the deprecation notice in the entry? I feel like it makes this more clear. Something like:

```suggestion
acl: Deprecate the `token` query parameter and warn when it is used for authentication.
```